continuous_integration/environment-3.6.yaml

```diff
@@ -22,7 +22,7 @@ dependencies:
   - zarr
   - tiledb-py
   - sqlalchemy
-  - pyarrow>=0.14.0,!=2.0.0
+  - pyarrow=0.14.*
```

Comment:
I pinned this to 0.14, because that is what in practice for all platforms gets used (and by putting it in the env file, it's a bit easier to see this)